packages/vite-node/src/utils.ts

```diff
@@ -8,6 +8,10 @@ export function slash(str: string) {
   return str.replace(/\\/g, '/')
 }
 
+export function mergeSlashes(str: string) {
```

Comment:
we already have this function in `utils/base.ts`

Reply:
Oh, I didn't notice. I guess no, we shouldn't. Maybe sometime we can create a utils package, but not in this issue, I think